pkg/manila/funcs.go

```diff
@@ -42,6 +42,26 @@ func GetManilaSecurityContext() *corev1.SecurityContext {
 	}
 }
 
+// dbSyncSecurityContext - currently used to make sure we don't run db-sync as
+// root user
+func dbSyncSecurityContext() *corev1.SecurityContext {
+	runAsUser := ManilaUserID
+	runAsGroup := ManilaGroupID
+
+	return &corev1.SecurityContext{
+		RunAsUser:  &runAsUser,
+		RunAsGroup: &runAsGroup,
+		Capabilities: &corev1.Capabilities{
+			Drop: []corev1.Capability{
+				"MKNOD",
```

Comment:
Did dropping "ALL" fail here? I wonder what Caps are needed..

Reply:
Yes, I tried added both `SETUID` and `SETGID` caps (and removing `ALL`), however `kolla_start` fails to run the init script.
One thing we can do, as an alternative solution, is to remove `kolla` and run the command `/usr/bin/manila-manage --config-dir /etc/manila/manila.conf.d db sync" directly. This way we shouldn't need any additional capability and we can `Drop: All`.
I'm going to update the patch and try that.